src/main/scala/pureconfig/ConfigConvert.scala

```diff
@@ -250,5 +250,23 @@ object ConfigConvert {
       cc.value.to(gen.to(t), namespace)
     }
   }
+}
 
+/**
+ * A ConfigConvert for Duration defined in a trait to give it lower priority.
+ */
+trait LowPriorityConfigConvertImplicits {
+  import scala.concurrent.duration.Duration
+  implicit val durationConfigConvert: ConfigConvert[Duration] = new ConfigConvert[Duration] {
+    override def from(config: RawConfig, namespace: String): Try[Duration] = {
+      config.get(namespace).fold[Try[Duration]](Failure(new Exception(s"Couldn't read config key $namespace."))) { stringyDuration =>
```

Comment:
I was simply trying to use the typical English "<adjective> <noun>" construction. I've renamed all occurrences to `durationString`.